src/raftstore/store/peer.rs

```diff
@@ -962,6 +973,7 @@ impl Peer {
             // leader needs to update lease and last commited split index.
             let mut lease_to_be_updated = self.is_leader();
             let mut split_to_be_updated = self.is_leader();
+            let mut merge_to_be_update = self.is_leader();
```

Comment:
Seems `lease_to_be_updated`, `split_to_be_updated` and `merge_to_be_updated` are always the same, can we combine them together?

Reply:
Actually they are different, `lease_to_be_updated` depends on `Peer.ProposalQueue`, `split_to_be_updated` depends on split commands and `merge_to_be_updated` depends on merge commands.